truffle/src/main/ruby/core/proc.rb

```diff
@@ -55,8 +55,8 @@ def curry(curried_arity = nil)
       raise ArgumentError, "cannot create binding from f proc"
     end
 
-    f.singleton_class.thunk_method :parameters, [[:rest]]
-    f.singleton_class.thunk_method :source_location, nil
+    # f.singleton_class.thunk_method :parameters, [[:rest]]
+    # f.singleton_class.thunk_method :source_location, nil
```

Comment:
If we don't need this code we should remove it. If it needs to go back in later to complete the rest of the specs then we should add a descriptive `todo` comment.